### PR TITLE
get cue temp from remote through URI

### PR DESCRIPTION
**Pull request by captainroy-hy**
When it's going to get cue templates of capabilities, use templateURI as higher priority than template strings in definition file. 

BTW, I will complete tests after this PR containing test data is merged.

### CODE REVIEW

File: pkg/plugins/cluster.go [hunk 1]
```diff
@@ -100,12 +101,28 @@ func HandleTemplate(in *runtime.RawExtension, name, syncDir string) (types.Capab
...
+		res, err := http.Get(tmp.CueTemplateURI)
+		if err != nil {
+			return types.Capability{}, err
+		}
+		defer res.Body.Close()
+		b, err := ioutil.ReadAll(res.Body)
+		if err != nil {
+			return types.Capability{}, err
+		}
+		cueTemplate = fmt.Sprintf("%s", b)
```
Comment (wonderflow) [inline pkg/plugins/cluster.go:116]:
```suggestion
		cueTemplate = string(b)
```

Comment (wonderflow) [inline pkg/plugins/cluster.go:116]: can you try add some test for this case

**Comment by wonderflow** [comment]
you could use existing file here to do test https://github.com/oam-dev/kubevela/blob/master/vela-templates/web-service.cue @captainroy-hy

**Comment by captainroy-hy** [comment]
> you could use existing file here to do test https://github.com/oam-dev/kubevela/blob/master/vela-templates/web-service.cue @captainroy-hy

Sure, got it.

**Review by wonderflow — APPROVED** [review]
Good job

**Pull request merged**